src/main/js/distribution/Index.js

```diff
@@ -184,6 +184,18 @@ class Index extends Component {
         return <IconTableCellFormatter handleButtonClicked={this.copyButtonClicked} currentRowSelected={row} buttonIconName="copy" buttonText="Copy" />;
     }
 
+    enabledState(cell, row) {
+        const icon = (cell == 'true') ? 'check' : 'times';
+        const color = (cell == 'true') ? 'green' : 'red';
```

Comment:
Do we have constants for these colors? I think we have slight variations with each to comply with Synopsys branding requirements.